server/env.js

```diff
@@ -1,4 +1,6 @@
-import 'babel-polyfill';
+import Promise from 'bluebird';
+
+global.Promise = Promise;
```

Comment:
More specifically about Bluebird.

Why do we need to declare it as a global? How was that handled before this PR?

Where are we relying on a Bluebird as the global Promise object?

Reply:
Before `babel-polyfill` was replacing Promise with Bluebird and `async` functions with the `regenerator-runtime` (https://babeljs.io/docs/en/v7-migration#remove-proposal-polyfills-in-babel-polyfill-https-githubcom-babel-babel-issues-8416). We should get to a place where we use native Promises but that is a larger project.